webscrambles/src/net/gnehzr/tnoodle/server/webscrambles/ScrambleRequest.java

```diff
@@ -797,19 +797,20 @@ private static TableAndHighlighting createTable(PdfWriter docWriter, Document do
         Font scrambleFont = null;
 
         try {
-            BaseFont courier = BaseFont.createFont(BaseFont.COURIER, BaseFont.CP1252, BaseFont.EMBEDDED);
+            // load the font from the resource folder. This loads the regular version, but can be changed to light, bold, etc if people prefer that
+            BaseFont sourcecode = BaseFont.createFont(Utils.getResourceDirectory().getPath() + "/fonts/SourceCodePro-Regular.ttf", BaseFont.CP1252, BaseFont.EMBEDDED);
```

Comment:
I know we previously called this variable `courier`, but I think we should call this `sourceCodeFont`. `courier` is obviously a font to me, whereas `sourcecode` has all sorts of other meanings =)